### DISCUSSION TITLE
2024/01/19/%E6%9C%AA%E6%88%90%E5%B9%B4%E7%94%B3%E8%AF%B7%E4%B8%AD%E4%BF%A1%E9%93%B6%E8%A1%8CVisa%E5%92%8CMasterCard%E5%80%9F%E8%AE%B0%E5%8D%A1/

### DISCUSSION THREAD

**Discussion by giscus** [bingxin666/blogmessages#3] [Announcements]
# 2024/01/19/%E6%9C%AA%E6%88%90%E5%B9%B4%E7%94%B3%E8%AF%B7%E4%B8%AD%E4%BF%A1%E9%93%B6%E8%A1%8CVisa%E5%92%8CMasterCard%E5%80%9F%E8%AE%B0%E5%8D%A1/

好久不见！考试前我通过朋友了解到中信银行有Visa和MasterCard的借记卡，于是便尝试申请。刚好考试也结束了，那就正好去激活一下卡片

https://blog.byteloid.one/2024/01/19/%E6%9C%AA%E6%88%90%E5%B9%B4%E7%94%B3%E8%AF%B7%E4%B8%AD%E4%BF%A1%E9%93%B6%E8%A1%8CVisa%E5%92%8CMasterCard%E5%80%9F%E8%AE%B0%E5%8D%A1/

**Comment by ltzXiaoYanMo** [comment]
Oracle那玩意是人过的吗（
